spark-operator-api/src/main/java/org/apache/spark/kubernetes/operator/BaseResource.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.spark.kubernetes.operator;
+
+import io.fabric8.kubernetes.api.model.Namespaced;
+import io.fabric8.kubernetes.client.CustomResource;
+
+import org.apache.spark.kubernetes.operator.spec.BaseSpec;
+import org.apache.spark.kubernetes.operator.status.BaseAttemptSummary;
+import org.apache.spark.kubernetes.operator.status.BaseState;
+import org.apache.spark.kubernetes.operator.status.BaseStatus;
+
+public class BaseResource<
+        S,
```

Comment:
indentation?

Reply:
This is introduced by spotless GoogleJavaStyle plugin. I have not yet dig into the implementation, but seems this could be related to the [guide](https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent) for 

> When line-wrapping, each line after the first (each continuation line) is indented at least +4 from the original line.

... and it attempts a +4+4 here. 

Spotless GoogleJavaStyle does not allow additional customization. If this line wrapping style becomes a concern we may switch to form our own style xml.